### PR TITLE
Documentation scripts updates needed for e102

### PR DESCRIPTION
**Pull request by ima23**
routine/ done every release:
- update of mwb and schema docs

new for e102:
- remove allele_code example
- source urls: ALFA, NESP
- add automation for phenotype class examples

fixes missing from e101:
- GGV url, VCF source dbs, 
- fix dog image - affected e102 new sheep, 
- remove exac from list

### CODE REVIEW

File: scripts/misc/sources2html.pl [hunk 1]
```diff
@@ -433,6 +449,30 @@ sub source_table {
   
   ########## Sources ##########    
   
+  # For variation dbs where data is based on VCF
+  if ($is_vcf ) {
+    #need Source name, version =-, Description  + data_types
+    # get vcf information:
+    my $sth3 = get_connection_and_query($db_name, $hostname, $sql_3);
+    my $vcf_sample = $sth3->fetchrow_array;
```
[the remark this answers is not in the document]
Comment (ima23) [reply scripts/misc/sources2html.pl:457]: Yes, we are.

File: scripts/misc/sources2html.pl [hunk 2]
```diff
@@ -365,19 +378,22 @@ sub source_table {
...
+  my $is_vcf       = shift;
   my $db_name      = shift;
   my $hostname     = shift;
   my $p_list       = shift;
   my $p_set_list   = shift;
 
   
   my @name_parts = split('_', $name);
-  if (@name_parts > 2) {
+  if (@name_parts > 3) {
```
[the remark this answers is not in the document]
Comment (ima23) [reply scripts/misc/sources2html.pl:389]: No, but we have with 3. This is a fix for dog and new sheep name issue.

**Review by aparton — APPROVED** [review]
Happy to merge. Does anything need to be done to the 103 PR or can I merge them both?

**Comment by ima23** [comment]
> Happy to merge. Does anything need to be done to the 103 PR or can I merge them both?

Thanks for checking @aparton. Both are ready for merge.

**Pull request merged**